serverless/integration/integration_test.go

```diff
@@ -201,3 +210,29 @@ func httpFetcher(t *testing.T, u string) client.FetcherFunc {
 		return ioutil.ReadAll(resp.Body)
 	}
 }
+
+func mustGetSigner(t *testing.T, privKey string) note.Signer {
+	s, err := note.NewSigner(privKey)
```

Comment:
nit: add a `t.Helper()` here (just helps the test framework accurately report where a problem is if there's a fail)

Reply:
Done